Issue1316

phkahler opened the pull request:
Closes #1316.  Fixes point H/V constraints. Fixes some undo/redo with constraints.

77maxikov commented:
I've lost one possible thing for multiple constraining - the case when several points coincide on https://github.com/solvespace/solvespace/blob/3d3d5c789d7035517bec6a4df6e7590f5f22e680/src/constraint.cpp#L274

phkahler commented:
> I've lost one possible thing for multiple constraining - the case when several points coincide on
> 
> https://github.com/solvespace/solvespace/blob/3d3d5c789d7035517bec6a4df6e7590f5f22e680/src/constraint.cpp#L274

I fixed that. It should be on master. I also fixed points H/V which got broken, and I extended it to multiple points.

Try master now.
